packages/ckeditor5-language/docs/features/language.md

```diff
@@ -24,6 +24,30 @@ There are other language-related CKEditor 5 features you may want to check:
 * {@link features/ui-language UI Language}  &ndash; Set the UI language.
 * {@link features/spelling-and-grammar-checking Spelling and grammar checking} &ndash; Employ multi-language spell check for flawless content.
 
+## Configuring available languages
+
+To modify the list of available languages displayed in the language dropdown, {@link module:core/editor/editorconfig~LanguageConfig language config} needs to be updated.
+
+Example below shows the configuration used for the {@link features/language#demo demo} above:
+
+```js
+ClassicEditor
+    .create( document.querySelector( '#editor' ), {
+		language: {
+			textPartLanguage: [
+				{ title: 'Arabic', languageCode: 'ar' },
+				{ title: 'French', languageCode: 'fr' },
+				{ title: 'Hebrew', languageCode: 'he' },
+				{ title: 'Spanish', languageCode: 'es' }
+			]
+		},
+    } )
+    .then( ... )
+    .catch( ... );
+```
+
+Read more in {@link module:core/editor/editorconfig~LanguageConfig#textPartLanguage textPartLanguage}.
```

Comment:
1. Always use the noun describing that is "behind" the link (e.g. docs article, method, or, like here, configuration option).
2. We have a convention to prefix config option references with context, like in https://github.com/ckeditor/ckeditor5/blob/master/packages/ckeditor5-link/docs/features/link.md?plain=1#L82. So here we should have `config.language.textPartLanguage` (also - usually mind the backticks when quoting the code).